libs/sdk-ui-charts/src/interfaces/chartConfig.ts

```diff
@@ -189,6 +189,14 @@ export interface IChartConfig {
      * @internal
      */
     type?: VisType;
+
+    /**
+     * Allow zooming and panning in a chart.
+     * When this option is enabled, we will prepare some configs for the 'chart' option to allow the chart can be zooming/panning.
+     *
+     * @internal
```

Comment:
now when the zoomInsight is here as first class citizen, it is no longer needed to mark it as @internal. you can move it up to the 'public section' above the internals and remove the tag. it is the `chart` prop and the direct use of highchars which is internal (for historical reasons).

do I understand correctly that we want to 'advertise' this new zooming/panning capability? if that is true, have it as public.

Reply:
Yes, you right. I want to 'advertise' this new zooming/panning capability for the Highcharts. I have updated this file. Please have a look again. Thanks!!